src/Selenium2Driver.php

```diff
@@ -222,25 +222,29 @@ protected function withSyn()
     /**
      * Creates some options for key events
      *
-     * @param string $char     the character or code
-     * @param string $modifier one of 'shift', 'alt', 'ctrl' or 'meta'
+     * @param string|int  $char     the character or code
```

Comment:
why adding support for integers there ? This looks like a new feature

Reply:
This method is protected and not tagged as `@internal`. So someone extending the class (and I know some people do extend this class) could rely on it.

Reply:
https://github.com/minkphp/MinkSelenium2Driver/pull/358#discussion_r997034281 resolved, the phpdoc is changed correctly and `int` can/is passed by the testing suite

Reply:
if integers are passed without being handled explicitly here, how did it work before ? Were we dispatching broken events (in which case it mean that this is not _actually_ tested by the driver testsuite as it was passing) or was the old version of Syn fixing this for us ?

Reply:
@stof , I can confirm, that:

* `\Behat\Mink\Driver\DriverInterface::keyPress` method PhpDoc has `@param string|int $char     ...`;
* implementation of that interface method `\Behat\Mink\Driver\Selenium2Driver::keyPress` is passing that `$char` parameter to the `\Behat\Mink\Driver\Selenium2Driver::charToOptions` method (changed below).

I agree with the made PhpDoc change, but I don't agree with the `is_numeric` change into `is_int` as this could be a BC break.

Reply:
After a thorough investigation, I longer consider this a BC break and agree with a merge.